modules/cli/src/main/scala/scala/cli/commands/run/RunOptions.scala

```diff
@@ -3,16 +3,26 @@ package scala.cli.commands.run
 import caseapp.*
 import caseapp.core.help.Help
 
-import scala.cli.commands.shared.{HasSharedOptions, SharedOptions}
+import scala.cli.ScalaCli
+import scala.cli.commands.shared.{HasSharedOptions, HelpMessages, SharedOptions}
 
 // format: off
-@HelpMessage("""|Compile and run Scala code.
-                |
-                |To pass arguments to the application, just add them after `--`, like:
-                |
-                |```sh
-                |scala-cli MyApp.scala -- first-arg second-arg
-                |```""".stripMargin)
+@HelpMessage(s"""|Compile and run Scala code.
+                 |
+                 |${HelpMessages.commandConfigurations("run")}
+                 |
+                 |For a run to be successful, a main method must be present on the classpath.
+                 |.sc scripts are an exception, as a main class is provided in their wrapper.
+                 |
+                 |${HelpMessages.acceptedInputs}
+                 |
+                 |To pass arguments to the actual application, just add them after `--`, like:
+                 |
+                 |```sh
+                 |${ScalaCli.progName} run Main.scala AnotherSource.scala -- first-arg second-arg
```

Comment:
```suggestion
                |${ScalaCli.progName}  Main.scala AnotherSource.scala -- first-arg second-arg
```
Do we want to suggest here running `run` command? as it is default command

Reply:
it is the help for the `run` sub-command though... so I'd rather not point to the default sub-command here, if only to avoid confusion